client/my-sites/checkout/composite-checkout/test/checkout-main.tsx

```diff
@@ -83,12 +84,15 @@ describe( 'CheckoutMain', () => {
 			additionalCartProps: Partial< Parameters< typeof ShoppingCartProvider > >;
 			useUndefinedCartKey?: boolean;
 		} ) => {
+			console.log( `additionalProps: ${ JSON.stringify( additionalProps ) }` );
```

Comment:
I think this console.log is a leftover from local tests

Reply:
Ooh, nice catch, thanks!